packages/mobile/src/App.tsx

```diff
@@ -58,6 +58,19 @@ const App = () => {
     setIsReadyToSetupBackend(!entropy)
   }, [])
 
+  useEffectOnce(() => {
```

Comment:
I think this is fine, but maybe we could add this into a saga? I think we do a similar thing with the mobile theme saga to avoid having one-off effects in the root?

Reply:
Well we do have the entropy / backend setup effect here too. I kinda prefer keeping things out of sagas but either way works